tiledb/sm/buffer/buffer.cc

```diff
@@ -42,49 +41,120 @@ using namespace tiledb::common;
 namespace tiledb {
 namespace sm {
 
-/* ****************************** */
-/*             MACROS             */
-/* ****************************** */
+// =======================================================
```

Comment:
NIT: maybe switch those to the following format as we use it other places in the code?
/* ****************************** */
/*   BufferBase                               */
/* ****************************** */

Reply:
I find lines made of equal signs far faster to see section breaks that asterisks for some reason. So I switched to my default on that.

But you gave me an idea. I haven't thought about this in quite a while. Not since before Unicode was available. How about we switch to using U+2500 Box Drawings Light Horizontal. It works even better as a rapid visual signal.

```
//───────────────────────────────────────────────────────
// BufferBase
//───────────────────────────────────────────────────────

```

Reply:
Sure I like those better too for a variety of reasons... I was just advocating for consistency to not have two style boxes in the code. But then again there's less than 150 in the entire code base so I could go both ways :)

Reply:
Not sure my favored editor does unicode, 
what flavor of unicode (unices seem to only like utf8, windows maybe not so much?)
and what about (editors of) the general public (or others using tiledb)?

Reply:
UTF8 has become completely non-problematic because it's the de facto standard for HTML encoding. Please check your editor, but I doubt there's a problem.

Reply:
I am fine either way, as long as we are consistent everywhere.